src/frontend/apps/desk/src/pages/404.tsx

```diff
@@ -1,25 +1,72 @@
-import { ReactElement } from 'react';
-import { useTranslation } from 'react-i18next';
+import {Button} from '@openfun/cunningham-react';
+import {ReactElement} from 'react';
+import {useTranslation} from 'react-i18next';
 
-import { Text } from '@/components';
-import { NextPageWithLayout } from '@/types/next';
+import {Box, Text, StyledLink} from '@/components';
+import {NextPageWithLayout} from '@/types/next';
 
 import MainLayout from './MainLayout';
 
+import Icon404 from '@/assets/icons/icon-404.svg';
+
+import styled from "styled-components";
+import useCunninghamTheme from "@/cunningham/useCunninghamTheme";
+
+const StyledButton = styled(Button)`
+  width: fit-content;
+  padding-left: 2rem;
+  padding-right: 2rem;
+`;
+
 const Page: NextPageWithLayout = () => {
-  const { t } = useTranslation();
+  const {t} = useTranslation();
+  const {colorsTokens} = useCunninghamTheme()
 
   return (
-    <Text
-      className="mt-xl"
-      as="h1"
-      $justify="center"
+    <Box
       $align="center"
-      $theme="danger"
-      $textAlign="center"
+      className="mr-s ml-s"
+      $css={'margin-top: 115px'}
```

Comment:
By pushing from the top with this kind of unit, you loose responsiveness.
What about that:
```suggestion
      className="m-auto"
      $height="70vh"
      $gap="2rem"
```
Prefer `gap` in your parent than `margin-top` for your children element, same because it helps to keep things a bit responsive.